<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>601</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="PersonalInfoTab">
     <attribute name="title">
      <string>Personal Info</string>
     </attribute>
     <widget class="QLabel" name="NameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="AgeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Of Birth</string>
      </property>
     </widget>
     <widget class="QLabel" name="GenderLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QComboBox" name="GenderComboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="NameEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>200</width>
        <height>25</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;User&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QDateEdit" name="AgeEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HealthInfoTab">
     <attribute name="title">
      <string>Health Info</string>
     </attribute>
     <widget class="QLabel" name="WeightLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Weight</string>
      </property>
     </widget>
     <widget class="QLabel" name="HeightLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="WeightEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;80&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="HeightEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;180&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="MeasuringUnitLabel1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Kg</string>
      </property>
     </widget>
     <widget class="QLabel" name="MeasuringUnitLabel2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
     <widget class="QLabel" name="DiseasesLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Disease</string>
      </property>
     </widget>
     <widget class="QComboBox" name="DiseasesComboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diabetes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tooth Decay</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ulcer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Food Allergies</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cardiac Diseases</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Anemia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inflammatory Bowel Disease</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Irritable Bowel Syndrome</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="PressureLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Blood Pressure</string>
      </property>
     </widget>
     <widget class="QLabel" name="BloodType">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Blood Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="BloodTypeComboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>A+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>A-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>O-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AB+</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AB-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="PressureEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;120/80&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="ExerciseLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exercise days</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ExerciseDaysComboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="SubmitPushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelPushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPatient_Information_System">
    <property name="title">
     <string>Nutrition System</string>
    </property>
   </widget>
   <addaction name="menuPatient_Information_System"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
